bin/install_windows_10_sdk.ps1

```diff
@@ -1,18 +1,41 @@
+param ( # parameters need to be defined at the top of the script
+  [switch]$DryRun = $false
+)
+
 # Stop script execution when a non-terminating error occurs
 $ErrorActionPreference = "Stop"
 
-Write-Host "--- :windows: Installing Windows 10 SDK and Visual Studio Build Tools"
+Write-Output "--- :windows: Installing Windows 10 SDK and Visual Studio Build Tools"
 
 $windowsSDKVersionFile = ".windows-10-sdk-version"
 if (-not (Test-Path $windowsSDKVersionFile)) {
   Write-Output "[!] No Windows 10 SDK version file found at $windowsSDKVersionFile."
   exit 1
 }
 
-$windows10SDKVersion = Get-Content $windowsSDKVersionFile
+$windows10SDKVersion = Get-Content $windowsSDKVersionFile `
+  | Where-Object { $_ -match '\S' }  # Remove empty lines
+
+if ($windows10SDKVersion.Count -ne 1) {
+  Write-Output "[!] Invalid version file format."
+  Write-Output "Expected exactly one non-empty line, got:"
+  Write-Output ($windows10SDKVersion -join "`n")  # The join poperly formats multiple lines
```

Comment:
```suggestion
  Write-Output ($windows10SDKVersion -join "`n")  # The join properly formats multiple lines
```

Reply:
Before accepting this suggestion, I'll test [the one above](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/153#discussion_r1961399525), because if it works I'd say we can remove this check altogether.

I mean, it might be good to let the user know the added rubbish in `.windows-10-sdk-version` but  it might also be an annoying build failure given we were able to parse it anyway.